packages/react-native/React/Views/RCTModalHostView.m

```diff
@@ -173,6 +186,7 @@ - (void)ensurePresentedOnlyIfNeeded
     RCTAssert(self.reactViewController, @"Can't present modal view controller without a presenting view controller");
 
     _modalViewController.supportedInterfaceOrientations = [self supportedOrientationsMask];
+    _modalViewController.modalInPresentation = !self.interactiveDismissal;
```

Comment:
`_modalViewController` is set in the `initWithBridge` of this class where it sets by default
```obj-c
#if defined(__IPHONE_OS_VERSION_MAX_ALLOWED) && defined(__IPHONE_13_0) && \
    __IPHONE_OS_VERSION_MAX_ALLOWED >= __IPHONE_13_0
  if (@available(iOS 13.0, *)) {
    self.modalInPresentation = YES;
  }
#endif
```
Shouldn't that be removed and be set here with the same condition?

Reply:
Yes, this seems like a reasonable condition, I'll add this.

Reply:
Actually @Gregoirevda can you point me to where that is defined?

Couldn't find the condition in `packages/react-native/React/Views/RCTModalHostViewController.m` or `packages/react-native/React/Views/RCTModalHostView.m`.

Reply:
@Gregoirevda it was removed when the minimum supported iOS version for RN was [bumped to 13.4 here](https://github.com/facebook/react-native/commit/610b14e4f3f606cd4c49518b9a42e3290fd52aeb)